indico/htdocs/js/indico/modules/abstracts.js

```diff
@@ -37,4 +37,67 @@
 
         setupListGenerator(filterConfig);
     };
+
+    global.setupAbstractPage = function setupAbstractPage() {
+        $('body').on('click', '.judge-button', function(evt) {
+            var $box = $('#abstract-decision-box');
+            var $this = $(this);
+            var $form = $box.find('form');
+
+            evt.preventDefault();
+
+            $.ajax({
+                url: $this.data('href'),
+                method: 'POST',
```

Comment:
Shouldn't happen when you `preventDefault()` the `indico:confirmed` event (sorry, forgot to mention that)